test/integration/plugins/nuagevsp/test_nuage_vsp_mngd_subnets.py

```diff
@@ -175,94 +207,148 @@ def test_01_nuage_mngd_subnets_isonw(self):
         zone1 = self.create_vsd_zone(domain1, "ZoneToBeConsumedByACS")
         subnet1 = self.create_vsd_subnet(zone1, "SubnetToBeConsumedByACS",
                                          "10.0.0.1/24")
+        # self.create_vsd_dhcp_option(subnet1, 6,
+        #                            ["10.0.0.2","128.251.10.25","4.4.4.4"])
```

Comment:
Here and at other places : remove (or uncomment, if applicable) commented lines  (but bottom-line, don't leave commented code in the source).